test/metrics_test/trial.py

```diff
@@ -2,9 +2,14 @@
 import nni
 
 if __name__ == '__main__':
-    hyper_params = nni.get_next_parameter()
-
+    nni.get_next_parameter()
     for i in range(10):
+        if i % 2 == 0:
```

Comment:
why we add these code?

Reply:
Except local mode, for other training services, metrics are printed into stdout log file with special MARK 'NNISDK_ME'.
the purpose of this code is to test whether following 2 cases can be handled correctly:
1.  Intermediate result is not at the begining of a line, such as:
`report intermediate result without end of line.NNISDK_MExxx`
2.  Intermediate result is  at the begining of a line, such as:
```
report intermediate result.
NNISDK_MExxx
```